CHANGELOG.md

```diff
@@ -46,20 +46,11 @@ of response and not the raw API endpoint response
   - Adds `PUT /executions` endpoint to update an execution
 - **CUMULUS-2592**
   - Adds logging when messages fail to be added to queue
-- **CUMULUS-2576**
-  - Adds `PUT /granules` API endpoint to update a granule
-  - Adds helper `updateGranule` to `@cumulus/api-client/granules`
-- **CUMULUS-2606**
-  - Adds `POST /granules/{granuleId}/executions` API endpoint to associate an execution with a granule
-  - Adds helper `associateExecutionWithGranule` to `@cumulus/api-client/granules`
-- **CUMULUS-2583**
-  - Adds `queued` as option for granule's `status` field
+- **CUMULUS-2000**
```

Comment:
Ah, nice catch [`b4b0436` (#2479)](https://github.com/nasa/cumulus/pull/2479/commits/b4b0436b1cff9c046c6ed1a1e094f1b0b41315d9)

Reply:
Sorry, one more thing: I think we need to add this note under the 9.1.0 BREAKING CHANGES in this branch, because it got missed somehow but is there on master

```
- `@cumulus/api-client/granules.getGranule` now returns the granule record from the GET `/granules/<granuleId>` endpoint, not the raw endpoint response
```

https://github.com/nasa/cumulus/blob/feature/rds-phase-2/CHANGELOG.md#v910-2021-06-03
https://github.com/nasa/cumulus/blob/master/CHANGELOG.md#v910-2021-06-03

Reply:
[`f39f33f` (#2479)](https://github.com/nasa/cumulus/pull/2479/commits/f39f33f1f789366f7a54308576682ea77d00575b)